.gitconfig

```diff
@@ -0,0 +1,17 @@
+[user]
+  name = Alex Faissal
+  email = alefaissal@gmail.com
+[diff]
+[color.diff]
+  meta = yellow
+  new = green reverse
+  whitespace = red reverse
+[color]
+    ui = auto
+[alias]
+    st = status
+    ci = commit --verbose
+    co = checkout
+    di = diff -w
+    dc = diff --cached -w
+    noff = merge --no-ff
```

Comment:
Please remove this file from your project. This file should be in your HOME dir. https://github.com/redacademy/wr-fall-2019/wiki/Git-Merge-Workflow